integrationtests/test_all_examples.py

```diff
@@ -346,11 +346,26 @@ def test_run(self):
                     )
                 )
 
-
+@requires_toolkit([ToolkitName.qt, ToolkitName.wx])
 class TestInteractExample(unittest.TestCase):
     """ Test examples with more interactions."""
 
-    @requires_toolkit([ToolkitName.qt, ToolkitName.wx])
+    def test_button_editor_simple_demo(self):
+        # Test ButtonEditor_simple_demo.py in examples/demo/Standard_Edtiors
+        filepath = os.path.join(
+            DEMO, "Standard_Editors", "ButtonEditor_simple_demo.py"
+        )
+        demo = load_demo(filepath, "demo")
+
+        tester = UITester()
+        with tester.create_ui(demo) as ui:
+            button = tester.find_by_name(ui, "my_button_trait")
+            for _ in range(5):
+                button.perform(command.MouseClick())
+            click_counter = tester.find_by_name(ui, "click_counter")
+            displayed_count = click_counter.inspect(query.DisplayedText())
```

Comment:
There is still the issue on OSX/qt where the click_counter does not actually display a change, however this test still passes... Seems a little misleading given the test, but I think the tester, and query.DisplayedText() are generally working how we want them to.